src/app/components/Sidebar/index.tsx

```diff
@@ -102,7 +102,7 @@ const SidebarHeader = (props: SidebarHeaderProps) => {
     >
       <Link to="/">
         <Box align="center" direction="row" gap="small">
-          <Avatar src={process.env.PUBLIC_URL + '/logo192.png'} size={sizeLogo[size]} />
+          <Avatar src="/logo192.png" size={sizeLogo[size]} />
```

Comment:
This seems like it shouldn't work. When bundling assets often get a hash filename. https://parceljs.org/recipes/image/#javascript says you need to wrap it into `new URL`

Reply:
Plugin wasn't enabled in dev as I was testing prod build mainly. Anyway, works for both envs now. Build is keeping original file names for PNG and it works similar to current react-script build were PNG files are handled by raw loader. Caching is handled by AWS cache invalidation.